spec/models/vm/retirement_management_spec.rb

```diff
@@ -157,4 +157,24 @@
 
     vm.raise_audit_event(event_name, message)
   end
+
+  it "reset retirement state in future" do
+    @vm.update_attributes(:retirement_state => 'retiring')
+    options = { :date => Date.today + 1}
+    @vm.retire(options)
+
+    @vm.reload
+    expect(@vm.retirement_state).to be_nil
```

Comment:
You can do this line a single line: `expect(@vm.reload.retirement_state).to be_nil`